test/sim/abilities/wonderguard.js

```diff
@@ -43,4 +43,12 @@ describe('Wonder Guard', function () {
 		battle.setPlayer('p2', {team: [{species: "Reshiram", ability: 'turboblaze', moves: ['fusionflare']}]});
 		assert.hurts(battle.p1.active[0], () => battle.makeChoices('move sleeptalk', 'move fusionflare'));
 	});
+
+	it('should allow first turn of Sky Drop but not the second', function () {
+		battle = common.createBattle();
+		battle.setPlayer('p1', {team: [{species: "Smeargle", ability: 'wonderguard', moves: ['howl']}]});
+		battle.setPlayer('p2', {team: [{species: "Aerodactyl", ability: 'owntempo', moves: ['skydrop']}]});
+		battle.makeChoices('move howl', 'move skydrop');
+		assert.fullHP(battle.p1.active[0]);
```

Comment:
You're not testing for the side effects of the Howl. You need to test that the Howl didn't happen (that Smeargle's Atk isn't +1).

Reply:
You're also not testing that the second turn isn't allowed: this is only a test for the first turn.